[BUG]: test_intersections tests failing on 24.08

### Version

24.08

### On which installation method(s) does this occur?

_No response_

### Describe the issue

I'm seeing these tests fail across PRs targeting `branch-24.08`:

```text
FAILED tests/basicpreds/test_intersections.py::test_one_pair - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_two_pairs - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_one_pair_with_overlap - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_two_pairs_with_intersect_and_overlap - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_one_pair_multilinestring - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_three_pairs_identical_has_ring - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
FAILED tests/basicpreds/test_intersections.py::test_three_pairs_identical_no_ring - AssertionError: Series Expected type <class 'pandas.core.series.Series'>, found <class 'pandas.core.indexes.base.Index'> instead
```

### Minimum reproducible example

Observed this in the following PRs:

* #1394
* #1393
* #1384

ref: https://github.com/rapidsai/cuspatial/pull/1393#issuecomment-2153334961

### Relevant log output

_No response_

### Environment details

In those failing PRs, I see these relevant versions:

```text
cudf                      24.08.00a106    cuda11_py39_240605_g20aa4442d2_106    rapidsai-nightly
...
geopandas                 0.14.4             pyhd8ed1ab_0    conda-forge
geopandas-base            0.14.4             pyha770c72_0    conda-forge
...
pandas                    2.2.2            py39hfc16268_1    conda-forge
...
pyarrow                   16.1.0           py39h8003fee_1    conda-forge
pyarrow-core              16.1.0          py39h176f5a7_1_cpu    conda-forge
```

I see the failures across all combinations of Python version, CUDA version, and architecture... and for both wheels and conda packages.

([example build logs](https://github.com/rapidsai/cuspatial/actions/runs/9392305018/job/25866740018?pr=1393))

### Other/Misc.

_No response_

--- a/python/cuspatial/cuspatial/tests/basicpreds/test_intersections.py
+++ b/python/cuspatial/cuspatial/tests/basicpreds/test_intersections.py
@@ -1,3 +1,5 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
 import geopandas as gpd
 import pandas as pd
 from geopandas.testing import assert_geoseries_equal
@@ -13,7 +15,9 @@ def run_test(s1, s2, expect_offset, expect_geom, expect_ids):
         cuspatial.from_geopandas(s1), cuspatial.from_geopandas(s2)
     )
 
-    assert_series_equal(expect_offset, offset.to_pandas(), check_dtype=False)
+    assert_series_equal(
+        expect_offset, pd.Series(offset.to_pandas()), check_dtype=False
+    )
     assert_geoseries_equal(expect_geom, geoms.to_geopandas())
     assert_frame_equal(expect_ids, ids.to_pandas())
 

